osu.Game/Users/UserStatistics.cs

```diff
@@ -26,6 +27,13 @@ public struct LevelInfo
             public int Progress;
         }
 
+        /// <remarks>
+        /// This must only be used when coming from condensed user responses (e.g. from <see cref="GetUsersRequest"/>), otherwise use <code>Ranks.Global</code>.
+        /// </remarks>
+        // todo: this should likely be moved to a separate UserStatisticsCompact class at some point.
```

Comment:
this whole comment is ..

have you checked web side how this is done? it sounds like there should be a separate class for the statistics in that new storage area. aka. your todo but not a todo and part of this PR?

Reply:
hmm, now that you've mentioned it, making a `UserStatisticsCompact` class would be at least simpler to do than making a `UserCompact`, will add one here.